ground/src/main/res/layout/nav_drawer_header.xml

```diff
@@ -62,17 +62,18 @@
         <TextView
           android:layout_height="wrap_content"
           android:layout_width="wrap_content"
-          android:textSize="22sp"
+          android:textSize="18pt"
```

Comment:
I think we should still be using `sp` here:
https://developer.android.com/training/multiscreen/screendensities